plugins/splunk-on-call/README.md

```diff
@@ -0,0 +1,128 @@
+# Splunk On-Call
+
+## Overview
+
+This plugin displays Splunk On-Call, formerly VictorOps, information about an entity.
+
+There is a way to trigger an new incident directly to specific users or/and specific teams.
+
+This plugin requires that entities are annotated with a team name. See more further down in this document.
+
+This plugin provides:
+
+- A list of incidents
+- A way to trigger a new incident to specific users or/and teams
+- A way to acknowledge/resolve an incident
+- Information details about the persons on-call
+
+## Setup instructions
+
+Install the plugin:
+
+```bash
+yarn add @backstage/plugin-splunk-on-call
+```
+
+Add it to the app in `plugins.ts`:
+
+```ts
+export { plugin as SplunkOnCall } from '@backstage/plugin-splunk-on-call';
+```
+
+Add it to the `EntityPage.ts`:
+
+```ts
+import {
+  isPluginApplicableToEntity as isSplunkOnCallAvailable,
+  SplunkOnCallCard,
+} from '@backstage/plugin-splunk-on-call';
+// ...
+{
+  isSplunkOnCallAvailable(entity) && (
+    <Grid item md={6}>
+      <SplunkOnCallCard entity={entity} />
+    </Grid>
+  );
+}
+```
+
+## Client configuration
+
+In order to be able to perform certain action (create-acknowledge-resolve an action), you need to provide the username of the user making the action.
+The user supplied must be a valid Splunk On-Call user and a member of your organization.
+
+In `app-config.yaml`:
+
+```yaml
+splunkOnCall:
+  username: <SPLUNK_ON_CALL_USERNAME>
```

Comment:
Tbh this pretty awkward but an alright stopgap. I think the optimal way to do this would be to implement SSO, but that'll require some additions to the auth backend, so eh 😅 . Let's call it potential future improvements :grin: